specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/datamigration.json

```diff
@@ -751,6 +800,163 @@
+        "x-ms-examples": {
+          "Tasks_CreateOrUpdate": { "$ref": "./examples/ServiceTasks_CreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/projectTask"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service task updated",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "201": {
+            "description": "Service task created",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./definitions/Common.json#/definitions/ApiError"
+            }
+          }
+        }
+      },
+      "get": {
+        "summary": "Get service task information",
+        "description": "The service tasks resource is a nested, proxy-only resource representing work performed by a DMS instance. The GET method retrieves information about a service task.",
+        "tags": [
+          "Service Task resource",
+          "Standard operation",
+          "GET"
+        ],
+        "operationId": "ServiceTasks_Get",
+        "x-ms-examples": {
+          "Tasks_Get": { "$ref": "./examples/ServiceTasks_Get.json" }
+        },
+        "parameters": [
+          {
+            "name": "$expand",
+            "description": "Expand the response",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A service task resource",
+            "schema": {
+              "$ref": "./definitions/Tasks.json#/definitions/ProjectTask"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./definitions/Common.json#/definitions/ApiError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete service task",
+        "description": "The service tasks resource is a nested, proxy-only resource representing work performed by a DMS instance. The DELETE method deletes a service task, canceling it first if it's running.",
+        "tags": [
+          "Service Task resource",
+          "Standard operation",
+          "DELETE"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/deleteRunningTasks"
+          }
+        ],
+        "operationId": "ServiceTasks_Delete",
+        "x-ms-examples": {
+          "Tasks_Delete": { "$ref": "./examples/ServiceTasks_Delete.json" }
+        },
+        "responses": {
+          "200": {
+            "description": "Service Task resource deleted"
+          },
+          "204": {
+            "description": "Not found"
+          },
+          "default": {
```

Comment:
Please check if this is a long running operation

Reply:
Yes, same as project task.  We queue up the work which gets picked up by a worker.